### PR TITLE
Update query params and fix formatting

### PR DESCRIPTION
**Pull request by dosvath**
## Problem

Issues in https://github.com/Mentum-Inc/api_docs_av/pull/14

## Solution

Fixes from https://github.com/Mentum-Inc/api_docs_av/pull/14

## Testing Done

- [X] Spec builds using `redoc-cli serve`
- [X] Previewed changes locally

### CODE REVIEW

**Review by ncassata — APPROVED** [review]
LGTM

File: openapi/openapi.yml [hunk 1]
```diff
@@ -269,7 +269,7 @@ tags:
 # Reusable, common responses for endpoints
 components:
   securitySchemes:
-    Basic Authentication:
+    Basic_Authentication:
```
Comment (estevaofay) [inline openapi/openapi.yml:272]: I saw you changing this to underscore in the other PR also, any reason for it?

I'm pretty sure it works as separate words

**Review by estevaofay — APPROVED** [review]
LGTM

File: openapi/openapi.yml (hunk 1 above)
Comment (dosvath) [reply openapi/openapi.yml:272]: Readme OAS validation does not allow spaces in component names.

**Pull request merged**